stac_fastapi/extensions/stac_fastapi/extensions/core/filter.py

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+"""Filter Extension."""
+
+from typing import List
+
+import attr
+from fastapi import APIRouter, FastAPI
+
+from stac_fastapi.api.models import CollectionUri, EmptyRequest
+from stac_fastapi.api.routes import create_endpoint
+from stac_fastapi.types.core import BaseFiltersClient
+from stac_fastapi.types.extension import ApiExtension
+
+
+@attr.s
+class FilterExtension(ApiExtension):
+    """Filter Extension.
+
+    The filter extension adds several endpoints which allow the retrieval of queryables and
+    provides an expressive mechanism for searching based on Item Attributes:
+        GET /queryables
+        GET /collections/{collectionId}/queryables
+
+    https://github.com/radiantearth/stac-api-spec/blob/master/fragments/filter/README.md
+
+    Attributes:
+        client: Queryables endpoint logic
+        conformance_classes: Conformance classes provided by the extension
+
+    """
+
+    client: BaseFiltersClient = attr.ib()
+    conformance_classes: List[str] = attr.ib(
+        default=[
+            "https://api.stacspec.org/v1.0.0-beta.2/item-search#filter",
+            "https://api.stacspec.org/v1.0.0-beta.2/item-search#filter:simple-cql",
```

Comment:
FYI, these are about to change. https://github.com/radiantearth/stac-api-spec/pull/163

Reply:
Have changed the conformance classes to:

```python
        default=[
            "https://api.stacspec.org/v1.0.0-beta.2/item-search#filter:basic-cql",
            "https://api.stacspec.org/v1.0.0-beta.2/item-search#filter:item-search-filter",
            "https://api.stacspec.org/v1.0.0-beta.2/item-search#filter:cql-text",
        ]
```

this was my interpretation of the PR where the basic spatial and temporal operators were recommended. I added `cql-text` as default, as item-search must implement `GET` search. It would follow that the base implementation of the filters extension would also be a `GET` operation using `cql-text`

Reply:
Sorry, I was actually just trying to let you know that they were going to change for beta.3. For beta.2, it should still use `simple-cql`

Reply:
oh, ok. will revert.